src/main/java/mate/academy/intro/model/Book.java

```diff
@@ -36,4 +43,13 @@ public class Book {
     private String coverImage;
     @Column(nullable = false, columnDefinition = "TINYINT")
     private boolean isDeleted = false;
+    @ManyToMany(fetch = FetchType.LAZY)

```

Comment:
```suggestion
    @ManyToMany
```
It’s  default behavior